elements/pfe-band/src/pfe-band.js

```diff
@@ -61,6 +61,12 @@ class PfeBand extends PFElement {
 
   constructor() {
     super(PfeBand, { type: PfeBand.PfeType });
+
+    this._observer = new MutationObserver(() => {
```

Comment:
@kylebuch8 The _mapSchemaToSlots is done by pfelement, can you help me with why a mutation observer would be needed in the component?  Wouldn't that mean every component needs this and it should be moved to pfelement instead?

Reply:
I mean I'm looking at the commit history and I added it but I'm asking you because I've no idea why. 🙂